tools/docker/substrate-all-in-one/hooks/post_push

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
@RafaelAPB Please delete this and the directory it is in, we don't use DockerHub anymore (and this is specific for that)

What we need instead is a yaml file within the .github/workflows directory.
See the other yaml files of other container images there to get an idea of how it's done. (spoiler: 99% copy & paste)